libkbfs/folder_update_prepper.go

```diff
@@ -379,7 +387,8 @@ func (fup folderUpdatePrepper) prepTree(ctx context.Context, lState *lockState,
 		var childBps *blockPutState
 		// For an indirect file block, make sure a new
 		// reference is made for every child block.
-		if entryType != Dir && fblock.IsInd {
+		if flags&prepFolderCopyIndirectFileBlocks != 0 &&
```

Comment:
CR requires us to copy file blocks, because they might be in use in some other file on the merged branch.  We don't have that issue with SyncAll, so avoid making copies.